### ISSUE TITLE
Trouble while generating Elm file from Fluent files

### ISSUE THREAD

**Issue by purce** [andreasewering/travelm-agency#132]
I'm trying to generate the Elm file from two Fluent files, but the Elm file is empty.

```
npx travelm-agency --elm_path=src/I18n/I18n.elm --json_path=src/lang src/lang
Unable to parse file <STDIN>:1:69 To see a detailed explanation, run elm make on the file.
```

If I use JSON files as input, the Elm file is created successfully.

Am I missing something trivial? I'm not an npm and Elm expert. Is it something with intl-proxy?

**Comment by andreasewering** [comment]
Can you attach your .ftl files or copy paste their content in here so I can reproduce this locally?
Probably it failed to parse one of the ftl files in some way, but even if that is the case, I should be able to produce a better error message.

**Comment by purce** [comment]
It seems that it isn't able to handle identifier that contain dashes.

```
hello = Hello!
```
works.

```
hello-a = Hello!
```
doesn't work

```
-email = person@example.com

sendMail = Send a mail to { -email }
receiveMail = You received a mail from { -email }
```
works.

```
-email = person@example.com

send-Mail = Send a mail to { -email }
receiveMail = You received a mail from { -email }
```
doesn't work.

I played with the tutorial and the the identifiers with dashes are converted to functions whose names have dashes, but that isn't valid Elm. For example, `hello` and `hello-` should generate two functions. [The grammar rule of the identifier](https://github.com/projectfluent/fluent/blob/master/spec/fluent.ebnf#L85) accepts multiple dashes and underscores. How do you think it should be handled?

Moreover, I tried with this JSON:
```
{
    "hello": "Hello1",
    "hello-": "Hello2",
    "hello--": "Hello3"
}
```
and the generated code only handles the `hello--` key.

**Comment by andreasewering** [comment]
I'm going to make the assumption here that no one wants to have translation keys that only differ in the number of "-" that are postfixed. In the same way, I think "hello-world" and "helloWorld" should both just generate a function "helloWorld".
I just don't see a case where the distinction is useful, however both variants should compile.

So my suggested way to deal with this is converting kebab-case into camelCase when parsing. I should probably document this somewhere as well.

**Comment by andreasewering** [comment]
The issue should be fixed with the new version 3.1.1.
